Scala/Scala.sublime-syntax

```diff
@@ -440,16 +578,123 @@ contexts:
       push:
         - match: '`'
           pop: true
-    - match: '\b[a-z][a-zA-Z0-9_]*\.'
-    - match: '\.[a-z][a-zA-Z0-9_]*\b'
+    - match: '{{varid}}\.'
+    - match: '\.{{varid}}'
     - match: '\b{{varid}}'
       scope: entity.name.parameter.scala
+    - match: \[
+      push:
+        - match: \]
+          pop: true
+        - include: main
+    - match: '@|_'
```

Comment:
That doesn't sound like a keyword to me, just a "special" identifier. If you wanted to highlight it specially, I would expect a scope of `variable.language.scala` might be more accurate.